elytron/src/main/java/org/wildfly/extension/elytron/ElytronPersistentResourceXMLDescriptionFactory.java

```diff
@@ -0,0 +1,228 @@
+        PersistentResourceXMLDescription.PersistentResourceXMLBuilder builder = builder(ElytronExtension.SUBSYSTEM_PATH, schema.getNamespace());
+        if (schema.since(ElytronSubsystemSchema.VERSION_1_0)) {
+            builder.addAttribute(ElytronDefinition.DEFAULT_AUTHENTICATION_CONTEXT)
+                    .addAttribute(ElytronDefinition.INITIAL_PROVIDERS)
+                    .addAttribute(ElytronDefinition.FINAL_PROVIDERS)
+                    .addAttribute(ElytronDefinition.DISALLOWED_PROVIDERS)
+                    .addAttribute(ElytronDefinition.SECURITY_PROPERTIES, new AttributeParsers.PropertiesParser(null, SECURITY_PROPERTY, true), new AttributeMarshallers.PropertiesAttributeMarshaller(null, SECURITY_PROPERTY, true));
+        }
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_5_0)) {
+            builder.addAttribute(ElytronDefinition.REGISTER_JASPI_FACTORY)
+                    .addAttribute(ElytronDefinition.DEFAULT_SSL_CONTEXT);
+        }
+
+        addAuthenticationClientParser(builder, schema);
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_1_0)) {
+            builder.addChild(new ProviderParser().parser);
+        }
+
+        addAuditLoginParser(builder, schema);
+        addSecurityDomainParser(schema, builder);
+        addRealmParser(schema, builder);
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_1_0)) {
+            builder.addChild(new CredentialSecurityFactoryParser().parser);
+        }
+
+        addMapperParser(schema, builder);
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_3_0)) {
+            builder.addChild(getPermissionSetParser());
+        }
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_1_0)) {
+            builder.addChild(new HttpParser().parser);
+        }
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_1_0)) {
+            builder.addChild(new SaslParser().parser);
+        }
+
+        addTlsParser(schema, builder);
+        addCredentialStoreParser(schema, builder);
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_13_0)) {
+            builder.addChild(getExpressionResolverParser());
+        }
+
+        if (schema.since(ElytronSubsystemSchema.VERSION_3_0)) {
+            builder.addChild(getDirContextParser());
+        }
+
+        addPolicyParser(schema, builder);
+        if (schema.since(ElytronSubsystemSchema.VERSION_5_0)) {
+            builder.addChild(new JaspiConfigurationParser().jaspiConfigurationParser_5_0);
+        }
+
+        return builder.build();
+    }
+
+    private static void addPolicyParser(ElytronSubsystemSchema schema, PersistentResourceXMLDescription.PersistentResourceXMLBuilder builder) {
+        PolicyParser policyParser = new PolicyParser();
+        if (schema.since(ElytronSubsystemSchema.VERSION_1_2)) {
+            builder.addChild(policyParser.parser_1_2);
+        } else if (schema.since(ElytronSubsystemSchema.VERSION_1_1)) {
```

Comment:
Should this be schema.since(ElytronSubsystemSchema.VERSION_1_0)?

Reply:
Fixed